snapfaas/bins/admin_fstools/main.rs

```diff
@@ -27,9 +27,29 @@ pub fn main() -> std::io::Result<()> {
                 .args(&["bootstrap", "update_fsutil", "update_python", "faceted-list", "list", "read", "blob", "mkdir", "delete"])
                 .required(true),
         )
+        .arg(
+            Arg::with_name("tikv proxies")
+                .value_name("[ADDR:]PORT")
+                .long("tikv")
+                .takes_value(true)
+                .required(false)
+                .help("One or more addresses of TiKV placement driver, separated by space.")
+        )
         .get_matches();
 
-    let fs = snapfaas::fs::FS::new(&*snapfaas::fs::lmdb::DBENV);
+    let fs: FS<Box<dyn BackingStore>> = FS::new(
+        match matches.value_of("tikv proxies").map(|ts| ts.split_whitespace().into_iter().collect()) {
```

Comment:
There is a more idiomatic way to do this https://docs.rs/clap/2.33.0/clap/struct.Arg.html#method.multiple and potentially with https://docs.rs/clap/2.33.0/clap/struct.Arg.html#method.use_delimiter

Reply:
I think what we should spend some time doing is to bump clap version to 4.x.x
The current 2.33 is very old. For now, I'm gonna just merge this.

Reply:
Got it. Sounds good